### PR TITLE
Update on tool bar and headline section

### PR DESCRIPTION
**Pull request by yawsamcode**
Made the following updates to the mobile version of my portfolio: 

- Created toolbar/header
- Created a headline section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,253 @@
...
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+  npm intall -g live-server
```
Comment (hiromataba) [inline README.md:112]: - [ ] [OPTIONAL] Please fix the typo in this word: `npm intall -g live-server` to `npm install -g live-server`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            or website Look through some of my 
+            work and experience! If you like what you see and have a project 
+            you need coded, don’t hestiate to contact me.</p>
+
+        <ul class="header-social">
+            <li><a href="#"><img src="/images/github.svg" alt="GitHub" class="github"></a></li>
+            <li><a href="#"><img src="/images/Linkedin icon.svg" alt="Linkedin" class="linkedin"></a></li>
+            <li><a href="#"></a><img src="/images/angel list.svg" alt="Angel_List" class="angel-list"></li>
+            <li><a href="#"></a><img src="/images/twitter.svg" alt="Twitter" class="twitter"></a></li>
+            <li><a href="#"><img src="/images/medium.svg" alt="Medium" class="medium"></a></li>
```
Comment (hiromataba) [inline index.html:41]: - [ ] Please make the icons have the same size and leave some margin to match the original design 

- Your design 👇 

image


- Original design 👇 

image

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @yawsamcode    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (yawsamcode) [reply README.md]: Thank you for the feedback. I fixed it

File: index.html (hunk 2 above)
Comment (yawsamcode) [reply index.html:41]: Thank you for the feedback. I fixed it

File: README.md [hunk 3]
```diff
@@ -1 +1,252 @@
...
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
```
Comment (Kamba56) [inline README.md:28]: - [Optiinal] Kindly remove all comments from your readme file. The comments are just there to guide you on how to fill the sections.

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+  padding: 1px 24px;
+}
+
+.headline-section {
+  background-color: #1c1a19;
+  color: white;
+  background-image: url(/images/backgroundimage.svg);
```
Comment (Kamba56) [inline css/styles.css:47]: - [Optional] Your background image is not displaying, Kindly fix this, please.
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/styles.css">
+    <title>Sam | Full Stack Dev</title>
+</head>
+<body>
+    <div class="container">
+        <header class="nav-bar">
+                
+            <h1 class="name">Sam</h1>
```
Comment (Kamba56) [inline index.html:14]: - [Optional] Please note that you should have only one h1 element in your file. The h1 element serves as the title of your document.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    <h1>I’m Samuel <br></h1>
+        <h2>I’m a software developer!</h2>
+            <p>I can help you build a product , feature 
+            or website Look through some of my 
+            work and experience! If you like what you see and have a project 
+            you need coded, don’t hestiate to contact me.</p>
+
+        <ul class="header-social">
+            <li><a href="#"><img src="/images/github.svg" alt="GitHub" class="github"></a></li>
+            <li><a href="#"><img src="/images/Linkedin icon.svg" alt="Linkedin" class="linkedin"></a></li>
```
Comment (Kamba56) [inline index.html:38]: - [Optional] Please use a constant indentation of two spaces ina ll your documents, this is a convention and also best practice.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+                
+            <h1 class="name">Sam</h1>
+            <nav class="nav-links">
+                    <a href="#">Portfolio</a>
+                    <a href="#">About</a>
+                    <a href="#">Contact</a>
+            </nav>
+
+            <nav class="hamburger">
+            <img src="/images/Union.svg" alt="" class="menu">
```
Comment (Kamba56) [inline index.html:22]: - [Optional] Kindly wrap the hamburger in a button element as this is a button that when clicked, it opens the mobile menu.

**Review by Kamba56 — APPROVED** [review]
Hi @yawsamcode, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**